manifests/init.pp

```diff
@@ -14,12 +14,18 @@
 # [*node_path*]
 #   Value of the system environment variable (default: "/usr/local/node/node-default/lib/node_modules").
 #
-# [*python_package*]
-#   Python package name, defaults to python
-#
 # [*cpu_cores*]
 #   Number of CPU cores to use for compiling nodejs. Will be used for parallel 'make' jobs.
 #
+# [*contain_ruby*]
+#   Bool flag whether or not to install ruby.
+#
+# [*instances*]
+#   List of instances to install.
```

Comment:
furthermore the `nodejs-v6` is just for naming reasons in the YAML or puppet context. I rewrite them in an internal function to avoid accidental creation of duplicated resources.
(Imagine I'd create another resource with name `nodejs-v6` and this crashes with the code of someone, a lot of folks would complain about BC breaks)